### ISSUE TITLE
vggbn training's validation accuracy is very low

### ISSUE THREAD

**Issue by cxy7452** [soumith/imagenet-multiGPU.torch#63]
I was trying to train vggbn (16-layer, model 'D'), using batchsize = 24 with everything else in default setting, but the training does not converge at all, at the 5th iteration the training loss is still ~6.9.

then i tried adding nn.SpatialBatchNormalization after each convolutional layer and re-started the training, the training finally started to move. However, the validation accuracies are very low, these are my results for the first 8 iterations:

% top1 accuracy (test set) (center crop)    avg loss (test set) 
 2.8140e+00  5.9005e+00 
 8.1700e+00  4.9847e+00 
 1.1460e+01  4.6748e+00 
 1.4304e+01  4.3753e+00 
 1.8798e+01  3.9825e+00 
 2.2012e+01  3.7526e+00 
 2.2900e+01  3.7133e+00 

any thoughts..?

**Comment by cxy7452** [comment]
just tried the provided vanilla vgg, training does not converge at all, the training loss after 4 epochs is still at 6.9.

**Comment by arashno** [comment]
I had the same problem.
Adding BatchNorm after convolution layers helped me.
